src/app/dynamic/enterprise/metering/schedule-config/template.html

```diff
@@ -46,28 +46,30 @@
       <ng-container matColumnDef="name">
```

Comment:
https://github.com/kubermatic/dashboard/pull/4478/files#r884573276

Same as above for all
- matColumnDef -> `name, retention, interval, schedule, acions`